prowler/providers/aws/services/elbv2/elbv2_service.py

```diff
@@ -235,3 +343,13 @@ class LoadBalancerv2(BaseModel):
     listeners: list[Listenerv2]
     scheme: Optional[str]
     tags: Optional[list] = []
+    security_groups: list[str]
+
+
+class TargetGroups(BaseModel):
+    name: str
+    arn: str
+    target_type: str
+    target: str
+    public: bool
+    lbdns: Optional[str]
```

Comment:
Why you need the DNS of the load balancer in each target group?

Reply:
done. removed it